src/packages/client/src/__tests__/runtime-tests/aggregations/index.js

```diff
@@ -0,0 +1,47 @@
+const { PrismaClient } = require('@prisma/client')
+const assert = require('assert')
+
+module.exports = async () => {
+  const prisma = new PrismaClient()
+
+  const result = await prisma.user.aggregate({
+    avg: {
+      age: true,
+    },
+    count: true,
+    max: {
+      age: true,
+    },
+    min: {
+      age: true,
+    },
+  })
+
+  assert.deepEqual(result, {
+    avg: { age: 80 },
+    count: 10,
+    max: { age: 163 },
+    min: { age: 5 },
+  })
+
+  prisma.disconnect()
+}
+
+if (require.main === module) {
+  // seed()
+  module.exports()
+}
```

Comment:
@timsuchanek where is the data from for this test? the seed function is commented 🤔 (and how age can bepredictable when it's `age: Math.round(Math.random() * 200)` ?)